MapboxSearch/gradle.properties

```diff
@@ -21,7 +21,7 @@ android.enableJetifier=false
 kotlin.code.style=official
 
 # SDK version attributes
-VERSION_NAME=2.0.0-beta.3
+VERSION_NAME=2.0.0-location-updates-timeout2
```

Comment:
Is this expected version name? If you want to publish a snapshot, you should add `-SNAPSHOT` in the end, for example, `2.0.0-location-updates-timeout2-SNAPSHOT`

Reply:
Gotcha, next time I'll do it as you suggested. This is an experimental fix that we want our customer to try out.

Reply:
@DzmitryFomchyn It doesn't work, I'm reverting to my original naming approach.
```
Version 2.0.0-location-updates-timeout-SNAPSHOT is a SNAPSHOT. Skip publishing.

Exited with code exit status 1```